db-engine/src/main/java/com/splicemachine/db/iapi/stats/ColumnStatisticsImpl.java

```diff
@@ -111,6 +151,7 @@ public ColumnStatisticsImpl(DataValueDescriptor dvd,
     public void writeExternal(ObjectOutput out) throws IOException {
         out.writeLong(nullCount);
         out.writeObject(dvd);
+        out.writeInt(frequenciesSketchMaxMapSize);
```

Comment:
Is this the final version of the fix?  It doesn't look like the serialization issue is fixed yet.